toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -543,9 +544,10 @@ void RenameCellField::showInRowCol(int row, int col, bool multiColumnSelected) {
   setStyleSheet(styleSheetStr);
 
   TXshCell cell = xsh->getCell(row, col);
+  QPoint xy     = m_viewer->positionToXY(CellPosition(row, col)) - QPoint(1, 2);
   if (!cell.isEmpty()) {
     setFixedSize(XsheetGUI::ColumnWidth - 5, XsheetGUI::RowHeight + 4);
-    move(QPoint(m_viewer->columnToX(col) + 7, m_viewer->rowToY(row) - 2));
+    move(xy);
```

Comment:
Fixed.

Also fixed notes in cell area to resize to the same size as a single cell.